platform-api/src/services/cf_client.py

```
@@ -0,0 +1,160 @@
+        if resp.status_code != 200:
+            raise HTTPException(502, f"Cloudflare GraphQL API error: {resp.status_code}")
+
+        try:
+            # Parse the GraphQL response
+            response_json = resp.json()
+
+            # Check for GraphQL errors first
+            if "errors" in response_json:
+                error_details = response_json["errors"]
+                logger.error("GraphQL errors from Cloudflare Analytics API: %s", error_details)
+                raise HTTPException(502, f"GraphQL errors from Cloudflare Analytics API")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)